### DISCUSSION TITLE
Updating RancherOS 1.5.6 to Burmilla 1.9.1

### DISCUSSION THREAD

**Discussion by adityk** [burmilla/os#73] [General]
Hi All,

I have existing vm with rancheros 1.5.6 version and want to upgrade it with burmilla/os 1.9.1 version, but i am getting handshake error. Is there any specific changes i have to do on my existing vm.

SSH handshake err: ssh: handshake failed: read tcp 127.0.0.1:56230->127.0.0.1:22: read: connection reset by peer

**Comment by olljanat** [comment]
Hmm. Where you see that error? It does not look familiar.

**Reply by adityk** [reply]
this error get resolved once i increased the timeout for OS to come up, look like burmilla need more time to boot compare to rancher. 
The other query is does burmilla support xz compression, want to bundle the bumilla os as xz compressed file. will it be possible for burmilla.

**Reply by olljanat** [reply]
You increase timeout on where? How much CPU and RAM you have allocated for it?

What you mean with "want to bundle the bumilla os as xz compressed file. will it be possible for burmilla."? You are talking about to creating custom version of OS? What you are trying to achieve with that?

**Comment by pwFoo** [comment]
I updated my RancherOS to BurmillaOS and all works fine. No problems or additional steps needed. Just followed the instructions for migration (change path, ...)
VM is running on vmware. To be sure I created a snapshot before I migrate.
Some RancherOS internals should be cleaned up to (old user-*, kernels and initrd files).

I think important is the docker update. Some of my problems with RancherOS gone by migrating to Burmilla :)
